### PR TITLE
OSDOCS-11870-2: update xrefs MicroShift 4.17.y

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.17

Issue:
[OSDOCS-11870](https://issues.redhat.com/browse/OSDOCS-11870)

Link to docs preview:
[IPv6 link](https://81260--ocpdocs-pr.netlify.app/microshift/latest/microshift_release_notes/microshift-4-17-release-notes.html#microshift-4-17-IPv6_release-notes)
[Low latency and workload partitioning](https://81260--ocpdocs-pr.netlify.app/microshift/latest/microshift_release_notes/microshift-4-17-release-notes.html#microshift-4-17-running-apps_release-notes)

QE review:
- N/A links only, no technical changes

### CODE REVIEW

File: microshift_release_notes/microshift-4-17-release-notes.adoc [hunk 1]
```diff
@@ -49,8 +49,7 @@ Updating two minor versions in a single step is supported in 4.17. Updates for b
 
 [id="microshift-4-17-IPv6_{context}"]
 ==== IPv6 single and dual-stack IPv6 networking now available
-With this release, you can run {microshift-short} and your workloads with either IPv6 only or IPv4 and IPv6 dual stack networking protocols. Ingress and egress traffic can be on either IPv4 or IPv6 traffic, mixed or parallel. See xref:../microshift_configuring/microshift-using-config-tools.adoc#microshift-using-config-tools[Configuring IPv6 networking].
-//crossref to be updated
+With this release, you can run {microshift-short} and your workloads with either IPv6 only or IPv4 and IPv6 dual stack networking protocols. Ingress and egress traffic can be on either IPv4 or IPv6 traffic, mixed or parallel. See xref:../microshift_configuring/microshift-nw-ipv6-config.adoc#microshift-nw-ipv6-config[Configuring IPv6 networking].
```
Comment (tmalove) [inline microshift_release_notes/microshift-4-17-release-notes.adoc:52]:
```suggestion
With this release, you can run {microshift-short} and your workloads with either IPv6 only or IPv4 and IPv6 dual-stack networking protocols. Ingress and egress traffic can be on either IPv4 or IPv6 traffic, mixed or parallel. See xref:../microshift_configuring/microshift-nw-ipv6-config.adoc#microshift-nw-ipv6-config[Configuring IPv6 networking].
```
I checked some docs on the portal and saw the hyphenated version in some Quay documentation.

**Review by tmalove — APPROVED** [review]

File: microshift_release_notes/microshift-4-17-release-notes.adoc (hunk 1 above)
Comment (ShaunaDiaz) [reply microshift_release_notes/microshift-4-17-release-notes.adoc]: Agreed, it's a compound modifier here.

**Pull request merged**